### PR TITLE
Genre

### PR DESCRIPTION
**Pull request by VyacheslavBukach**

### CODE REVIEW

File: lib/presentation/screens/book_detail_screen.dart [hunk 1]
```diff
@@ -69,34 +75,64 @@ class BookDetailView extends StatelessWidget {
...
-            color: Colors.red,
-            onPressed: () {},
-            icon: const Icon(Icons.favorite),
-            iconSize: 60,
-          ),
+            Positioned(
+              right: 0,
+              bottom: 0,
+              child: StreamBuilder<bool>(
+                stream: getIt<CheckBookLikeUseCase>().checkBookLike(book.id),
```
Comment (MaximSemashko) [inline lib/presentation/screens/book_detail_screen.dart]: why do you use use case without bloc?

File: lib/presentation/screens/favorite_books_screen.dart [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            return SafeArea(
+              bottom: false,
+              child: Container(
+                padding: const EdgeInsets.only(
+                  left: 10,
+                  right: 10,
+                  top: 10,
+                ),
+                child: VerticalBookList(
+                  books: getIt<GetFavouriteBooksUseCase>().getFavouriteBooks(),
```
Comment (MaximSemashko) [inline lib/presentation/screens/favorite_books_screen.dart]: the same

File: lib/presentation/screens/store_screen.dart [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                padding: const EdgeInsets.only(
+                  left: 10,
+                  right: 10,
+                ),
+                child: BlocBuilder(
+                  bloc: BlocProvider.of<StoreBloc>(context),
+                  builder: (context, state) {
+                    if (state is DefaultStoreState) {
+                      return VerticalBookList(
+                        books: getIt<GetAllBooksUseCase>().getAllBooks(),
```
Comment (MaximSemashko) [inline lib/presentation/screens/store_screen.dart]: the same

File: lib/presentation/screens/store_screen.dart [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                      );
+                    }
+
+                    if (state is FilteredStoreState) {
+                      return VerticalBookList(
+                        books: getIt<GetFilteredBooks>()
```
Comment (MaximSemashko) [inline lib/presentation/screens/store_screen.dart]: the same

File: lib/presentation/screens/favorite_books_screen.dart (hunk 2 above)
Comment (VyacheslavBukach) [reply lib/presentation/screens/favorite_books_screen.dart]: done

File: lib/presentation/screens/book_detail_screen.dart (hunk 1 above)
Comment (VyacheslavBukach) [reply lib/presentation/screens/book_detail_screen.dart]: done

File: lib/presentation/screens/store_screen.dart (hunk 3 above)
Comment (VyacheslavBukach) [reply lib/presentation/screens/store_screen.dart]: done

File: lib/presentation/screens/store_screen.dart (hunk 4 above)
Comment (VyacheslavBukach) [reply lib/presentation/screens/store_screen.dart]: done

**Pull request merged**